workshop/Lab3/README.md

```diff
@@ -382,23 +405,27 @@ spec:
 ```
 
 - Create the service to access redis slaves.
-    ``` $ kubectl create -f redis-slave-service.yaml ```
+    ```shell
+    kubectl create -f redis-slave-service.yaml
+    ```
 
 - Restart guestbook so that it will find the slave service to read from.
     ```console
-    $ kubectl delete deploy guestbook-v1
-    $ kubectl create -f guestbook-deployment.yaml
+    kubectl delete deploy guestbook-v1
+    ```
+    ```shell
+    kubectl create -f guestbook-deployment.yaml
     ```
     
-- Test guestbook app using a browser of your choice using the url `<your-cluster-ip>:<node-port>`.
+- Test guestbook app using a browser of your choice using the url `<your-cluster-ip>:<node-port>`, or by simply refreshing the page if you have the app open in another window.
 
 That's the end of the lab. Now let's clean-up our environment:
 
 ```console
-$ kubectl delete -f guestbook-deployment.yaml
-$ kubectl delete -f guestbook-service.yaml
-$ kubectl delete -f redis-slave-service.yaml
-$ kubectl delete -f redis-slave-deployment.yaml 
-$ kubectl delete -f redis-master-service.yaml 
-$ kubectl delete -f redis-master-deployment.yaml
+kubectl delete -f guestbook-deployment.yaml
+kubectl delete -f guestbook-service.yaml
+kubectl delete -f redis-slave-service.yaml
+kubectl delete -f redis-slave-deployment.yaml 
+kubectl delete -f redis-master-service.yaml 
+kubectl delete -f redis-master-deployment.yaml
```

Comment:
this would certainly help in the "copy and paste it all in" gamers, is that the plan?

Reply:
yes